birdhouse/default.env

```diff
@@ -70,7 +72,7 @@ export CANARIE_MONITORING_EXTRA_CONF_DIR="/conf.d"
 export THREDDS_ORGANIZATION="Birdhouse"
 
 # Tag version that will be used to update Magpie API, Magpie CLI, and matching Twitcher with Magpie Adapter
-export MAGPIE_VERSION=3.26.0
+export MAGPIE_VERSION=3.31.0
```

Comment:
Ouch, I have to test Magpie DB upgrade.

Reply:
Ok as per https://github.com/bird-house/birdhouse-deploy/pull/283#pullrequestreview-1292276358